Makefile.ffmpeg

```diff
@@ -44,69 +44,72 @@ BSFS           = h264_mp4toannexb hevc_mp4toannexb
 FILTERS        = yadif format hwupload hwdownload scale null aresample anull
 HWACCELS       =
 
-NASM_VER       = 2.14.02
+NASM_VER       = 2.15.05
 NASM           = nasm-$(NASM_VER)
 NASM_TB        = $(NASM).tar.gz
 NASM_URL       = https://www.nasm.us/pub/nasm/releasebuilds/$(NASM_VER)/$(NASM_TB)
-NASM_SHA1      = bfa67d0d33f0c7b00c50d9f1bd33fd4d5df3c5d5
+NASM_SHA1      = 8a2a60b01d563c32dd216f94a99a20e6e31fd8aa
 NASM_DIFFS     = remove-invalid-pure_func-qualifiers.diff
 
-LIBX264        = x264-snapshot-20191216-2245
+LIBX264_VER    = db0d417728460c647ed4a847222a535b00d3dbcb
+LIBX264        = x264-$(LIBX264_VER)
 LIBX264_TB     = $(LIBX264).tar.bz2
-LIBX264_URL    = http://ftp.videolan.org/x264/snapshots/$(LIBX264_TB)
-LIBX264_SHA1   = 712f37aeabf3cb42f1ebe55588c33583636a7230
-
-LIBX265        = x265_3.2
-LIBX265_TB     = $(LIBX265).tar.gz
-LIBX265_URL    = http://ftp.videolan.org/videolan/x265/$(LIBX265_TB)
-LIBX265_SHA1   = 5922f88d0997c47e2c95feee95dc98bcb2e54871
+LIBX264_URL    = https://code.videolan.org/videolan/x264/-/archive/$(LIBX264_VER)/$(LIBX264_TB)
+LIBX264_SHA1   = 7b8da6f4283207b8ad77bab8890e3807f32013df
+
+LIBX265_VER    = 3.4
+LIBX265        = x265-$(LIBX265_VER)
+LIBX265_TB     = $(LIBX265_VER).tar.gz
+LIBX265_URL    = https://github.com/videolan/x265/archive/$(LIBX265_TB)
+LIBX265_SHA1   = eef65f2f9518f3396825675aa86bdf82a55aa0d9
 ifeq ($(CONFIG_PIE),yes)
 LIBX265_DIFFS  = libx265.pie.diff
 else
 LIBX265_DIFFS  = libx265.pic.diff
 endif
 
-LIBVPX_VER     = 1.8.2
+LIBVPX_VER     = 1.9.0-rc1
```

Comment:
there is a final 1.9.0

Reply:
Thanks for your review!

@fat-fred It would be great if you could change this.